src/Models/WebAuthnCredential.php

```diff
@@ -98,7 +98,7 @@ class WebAuthnCredential extends Model
     protected $visible = ['id', 'origin', 'alias', 'aaguid', 'attestation_format', 'disabled_at', 'is_enabled'];
 
     /**
-     * @return \Illuminate\Database\Eloquent\Relations\MorphTo&\Laragear\WebAuthn\Contracts\WebAuthnAuthenticatable
+     * @return \Illuminate\Database\Eloquent\Relations\MorphTo<\Laragear\WebAuthn\Contracts\WebAuthnAuthenticatable, \Laragear\WebAuthn\Models\WebAuthnCredential>
```

Comment:
AFAIK `MorphTo` needs two Models.
https://github.com/nunomaduro/larastan/blob/f72a8b1dfaa2eeaf746a93b3257591090c668794/stubs/MorphTo.stub#L6-L8

But `WebAuthnAuthenticatable` is not a Model.
What do you think?

Reply:
`WebAuthnAuthenticatable|Model`

Reply:
I seee!!
So it is a "future" model that implements `WebAuthnAuthenticatable`. We do not know its name :(